routing/pathfind.go

```diff
@@ -1211,7 +1209,9 @@ type blindedHop struct {
 // Any filtering of paths based on payment value or success probabilities is
 // left to the caller.
 func findBlindedPaths(g Graph, target route.Vertex,
-	restrictions *blindedPathRestrictions) ([][]blindedHop, error) {
+	restrictions *blindedPathRestrictions,
+	partiallySpecifiedPath []blindedpath.BlindedHop) (
+	[][]blindedpath.BlindedHop, error) {
 
 	// Sanity check the restrictions.
 	if restrictions.minNumHops > restrictions.maxNumHops {
```

Comment:
Old code but I think this check should be removed as well to the level where we set the restiction `rpcserver.go`

cc @ellemouton

Reply:
@ziggie1984  - but this is currently the first place we make use of `blindedPathRestrictions` after constructing it. 

Yes we should probably eventually merge `blindedPathRestrictions` and `BlindedPathRestrictions` and then do the checks in one place. But while this is the first usage of `blindedPathRestrictions`, it makes sense to have the check here i guess